### PR TITLE
UI test stubs for multi-net support feature

### PR DESCRIPTION
**Pull request by sghai**
Added test stubs for multi-net support feature.

### CODE REVIEW

File: tests/foreman/ui/test_multinetwork.py [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
+        subnet
+
+        2. All other fields on subnet page should be filled except start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Make sure to indent all text in a list. You can also drop the lines between list items.

```
1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
   subnet
2. All other fields on subnet page should be filled except start and
   end IP range
```

File: tests/foreman/ui/test_multinetwork.py [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+
+        @Assert: Host should be provisioned with correct configuration under
+         /etc/sysconfig/network-scripts/ifcfg-<interface>
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: This extra indentation is a syntax error.

File: tests/foreman/ui/test_multinetwork.py [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
+        subnet
+
+        2. All other fields on subnet page should be filled including start and
+        end IP range
+
+        @Assert: Host should be provisioned with correct configuration under
+         /etc/sysconfig/network-scripts/ifcfg-<interface>
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indentation.

File: tests/foreman/ui/test_multinetwork.py [hunk 4]
```diff
@@ -0,0 +1,717 @@
...
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
+        subnet
+
+        2. All other fields on subnet page should be filled including start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: List syntax.

File: tests/foreman/ui/test_multinetwork.py [hunk 5]
```diff
@@ -0,0 +1,717 @@
...
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set 'DHCP' for IPAM and BootMode for provisioning subnet
+
+        2. All other fields on subnet page should be filled
+
+        @Assert: Host should be provisioned with correct configuration under
+         /etc/sysconfig/network-scripts/ifcfg-<interface>
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indentation.

File: tests/foreman/ui/test_multinetwork.py [hunk 6]
```diff
@@ -0,0 +1,717 @@
...
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set 'DHCP' for IPAM and BootMode for provisioning subnet
+
+        2. All other fields on subnet page should be filled
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Can drop blank lines between list items.

**Comment by Ichimonji10** [comment]
The given comments apply in throughout the file. ACK pending fixes.

**Comment by sghai** [comment]
Ready for next round.. thanks for all review comments

File: tests/foreman/ui/test_multinetwork.py [hunk 7]
```diff
@@ -0,0 +1,588 @@
...
+        """@Test:  Delete an alias interface
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Create an aliased interface
+        2. Edit the new host -> Network --> delete the selected interface
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: The encoding of this file is declared as UTF-8, so you can use fancy characters like this if you want: →

Not necessary, of course. But it's slick. :)

File: tests/foreman/ui/test_multinetwork.py [hunk 8]
```diff
@@ -0,0 +1,588 @@
...
+        and BootMode set as 'DHCP' for provisioning subnet
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'None' and BootMode with 'DHCP' for provisioning
+        subnet
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

File: tests/foreman/ui/test_multinetwork.py [hunk 9]
```diff
@@ -0,0 +1,588 @@
...
+        and BootMode set as 'Static' for provisioning subnet
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'None' and BootMode with 'Static' for provisioning
+        subnet
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

File: tests/foreman/ui/test_multinetwork.py [hunk 10]
```diff
@@ -0,0 +1,588 @@
...
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'DHCP' and BootMode with 'Static' for provisioning
+        subnet
+        2. All other fields on subnet page should be filled except start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

File: tests/foreman/ui/test_multinetwork.py [hunk 11]
```diff
@@ -0,0 +1,588 @@
...
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'DHCP' and BootMode with 'Static' for provisioning
+        subnet
+        2. All other fields on subnet page should be filled including start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

File: tests/foreman/ui/test_multinetwork.py [hunk 12]
```diff
@@ -0,0 +1,588 @@
...
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'Static' for
+        provisioning subnet
+        2. All other fields on subnet page should be filled except start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

File: tests/foreman/ui/test_multinetwork.py [hunk 13]
```diff
@@ -0,0 +1,588 @@
...
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'Static' for
+        provisioning subnet
+        2. All other fields on subnet page should be filled including start and
+        end IP range
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_multinetwork.py]: Indent.

**Comment by Ichimonji10** [comment]
I've marked all the remaining syntax errors in the file. ACK pending fixes for them.

**Comment by sghai** [comment]
Thanks @Ichimonji10 for another review. Please take a look again.

**Comment by elyezer** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**